### ISSUE TITLE
Google Drive: Scope files to a specific root folder in My Drive / Team Drive

### ISSUE THREAD

**Issue by chrisedington** [onyx-dot-app/onyx#178]
It would be nice to be able to give the Google Drive connector a specific folder (whether under My Drive or a Shared Drive), as I'm sure there are certain folders in most cases that should not be indexed.

**Comment by yuhongsun96** [comment]
We're taking this one up! Thanks for creating a tracking issue

**Comment by chrisedington** [comment]
@yuhongsun96 thanks for adding it, saw it in the latest release :) It seems as though it's only for 'My Drive' and doesn't cater for 'Shared Drives'.

In terms of team usage, I think a significant amount of knowledge would be stored in a Shared Drive, where the content in personal drives is generally personal.

**Comment by yuhongsun96** [comment]
Yup, will be adding that as well! keeping the issue open in the meantime while we add that and also provide a smoother config experience for updating which folders to fetch from
